tests/cpp_api/audio_tests/README.md

```diff
@@ -0,0 +1,37 @@
+# rocAL Audio Unit Tests
+This application can be used to verify the functionality of the Audio APIs offered by rocAL.
+
+## Build Instructions
+
+### Pre-requisites
+* Ubuntu Linux, [version `20.04` or later](https://www.microsoft.com/software-download/windows10)
+* rocAL library (Part of the MIVisionX toolkit)
+* Radeon Performance Primitives (RPP)
+* MIVisionX
+* Sndfile
+
+### Build
+  ````
+  mkdir build
+  cd build
+  cmake ../
+  make
+  ````
+### Running the application
+  ````
+./audio_tests <audio-dataset-folder>
+
+Usage: ./audio_tests <audio-dataset-folder> <test_case> <downmix> <device-gpu=1/cpu=0> <qa_mode>
+  ````
+
+### Output verification 
+
+The python script `audio_tests.py` can be used to run all test cases for audio functionality in rocAL and verify the correctness of the generated outputs with the golden outputs.
+
+Input data is available in the following link : [MIVisionX-data](https://github.com/ROCm/MIVisionX-data/rocal_data)
```

Comment:
please be consistent with other tests and take path to `MIVisionX-data` and add rocal_data within script.